satispress.php

```diff
@@ -4,27 +4,31 @@
  *
  * @package SatisPress
  * @author Brady Vercher <brady@blazersix.com>
- * @license GPL-2.0+
+ * @license GPL-2.0-or-later
  *
  * @wordpress-plugin
  * Plugin Name: SatisPress
  * Plugin URI: https://github.com/blazersix/satispress
  * Description: Generate a Composer repository from installed WordPress plugins and themes.
- * Version: 0.2.3
+ * Version: 0.3.0-dev
  * Author: Blazer Six
  * Author URI: http://www.blazersix.com/
- * License: GPL-2.0+
+ * License: GPL-2.0-or-later
  * License URI: http://www.gnu.org/licenses/gpl-2.0.html
  * Text Domain: satispress
  * Domain Path: /languages
+ * GitHub Plugin URI: blazersix/satispress
+ * Requires PHP: 5.6
  */
 
+namespace SatisPress;
```

Comment:
@bradyvercher Are you happy with this namespace? I thought of doing `SatisPress\SatisPress` (as Vendor\Package, but it seemed redundant. This repo is the only result at https://packagist.org/?query=satispress